Classes/TextUtils.cs

```diff
@@ -24,16 +24,89 @@ namespace RadioDld
 
     public static class TextUtils
     {
+        // ## Key date formats to look out for
+        // 1 / d(dd)(st | nd | rd | th) | m(mm)(mmm) | yyyy('yy)(yy)
+        // 2 / yyyy('yy)(yy)|m(mm)(mmm)|d(dd)(st|nd|rd|th)
+        // 3 / mmm|d(dd)(st|nd|rd|th)|yyyy('yy)(yy)
```

Comment:
Is this block of comments useful to keep?  I'd think it is covered by the regex and tests..?

Reply:
I was thinking to keep, mostly for the 'next' time someone needs to review or revise what these regex statements are looking for.

Reply:
In that case, how about moving them to above each of the MatchFormat* consts?